firmware/user/display/cndraw.c

```diff
@@ -288,6 +288,268 @@ void ICACHE_FLASH_ATTR speedyWhiteLine( int16_t x0, int16_t y0, int16_t x1, int1
     }
 }
 
+
+
+
+/**
+ * @brief Optimized method to quickly draw a black line.
+ *
+ * @param x1, x0 Column of display, 0 is at the left
+ * @param y1, y0 Row of the display, 0 is at the top
+ *
+ */
+void ICACHE_FLASH_ATTR speedyBlackLine( int16_t x0, int16_t y0, int16_t x1, int16_t y1, bool thicc )
```

Comment:
Is this really much faster than adding a color arg? Seems like a lot of duplicated code between the black and white versions.